baggage-processor/src/test/java/io/opentelemetry/contrib/baggage/processor/BaggageProcessorCustomizerTest.java

```diff
@@ -38,57 +45,69 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 
 @ExtendWith(MockitoExtension.class)
-class BaggageSpanProcessorCustomizerTest {
+class BaggageProcessorCustomizerTest {
 
   private static final String MEMORY_EXPORTER = "memory";
 
   @Test
   void test_customizer() {
     assertCustomizer(Collections.emptyMap(), span -> span.hasTotalAttributeCount(0));
-    assertCustomizer(
-        Collections.singletonMap(
-            "otel.java.experimental.span-attributes.copy-from-baggage.include", "key"),
-        span -> span.hasAttribute(AttributeKey.stringKey("key"), "value"));
+    Map<String, String> properties = new HashMap<>();
+    properties.put("otel.java.experimental.span-attributes.copy-from-baggage.include", "key");
+    properties.put("otel.java.experimental.log-attributes.copy-from-baggage.include", "key");
+    // TODO try use
+    //  AttributeAssertion attributeAssertion =
+    // OpenTelemetryAssertions.equalTo(AttributeKey.stringKey("key"), "value");
+    assertCustomizer(properties, span -> span.hasAttribute(AttributeKey.stringKey("key"), "value"));
   }
 
   private static void assertCustomizer(
       Map<String, String> properties, Consumer<SpanDataAssert> spanDataAssertConsumer) {
 
     InMemorySpanExporter spanExporter = InMemorySpanExporter.create();
+    InMemoryLogRecordExporter logExporter = InMemoryLogRecordExporter.create();
 
-    OpenTelemetrySdk sdk = getOpenTelemetrySdk(properties, spanExporter);
+    OpenTelemetrySdk sdk = getOpenTelemetrySdk(properties, spanExporter, logExporter);
     try (Scope ignore = Baggage.current().toBuilder().put("key", "value").build().makeCurrent()) {
       sdk.getTracer("test").spanBuilder("test").startSpan().end();
+      sdk.getLogsBridge().get("test").logRecordBuilder().setBody("test").emit();
     }
+    // TODO verify log record attributes
     await()
         .atMost(Duration.ofSeconds(1))
         .untilAsserted(
-            () ->
-                TracesAssert.assertThat(spanExporter.getFinishedSpanItems())
-                    .hasTracesSatisfyingExactly(
-                        trace -> trace.hasSpansSatisfyingExactly(spanDataAssertConsumer)));
+            () -> {
+              TracesAssert.assertThat(spanExporter.getFinishedSpanItems())
+                  .hasTracesSatisfyingExactly(
+                      trace -> trace.hasSpansSatisfyingExactly(spanDataAssertConsumer));
+              List<LogRecordData> finishedLogRecordItems = logExporter.getFinishedLogRecordItems();
+                  assertThat(finishedLogRecordItems).hasSize(1);
```

Comment:
@zeitlinger is it enough testing of the log record?

Reply:
no, see last comment